cypress/integration/Lesson.spec.js

```diff
@@ -0,0 +1,34 @@
+describe("Lesson", () => {
+  beforeEach(() => {
+    cy.loadLessonFixtures();
+    cy.login();
+  });
+
+  it("Displays lesson contents", () => {
+    cy.get("@lessons").each(lesson => {
+      cy.log(lesson);
```

Comment:
is this a debug line or do we need this? Not super familiar with cypress

Reply:
No, it's not,  It can go.
```suggestion
```